cmd/cmd.go

```diff
@@ -39,6 +40,8 @@ import (
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
+const defaultConfigFilePath = "~/.config/stern/config.yaml"
```

Comment:
[The tilde expansion does not work](https://stackoverflow.com/questions/17609732/expand-tilde-to-home-directory) without the shell, so we might need to use [os.UserHomeDir()](https://pkg.go.dev/os#UserHomeDir).

Reply:
Oh... I'm going to use go-homedir `Expand`, since there seems to be a problem with "~" in `--config` flag and `STERNCONFIG` env.

Reply:
The shell expands the tilde before calling a program, so I believe we do not need to implement it for flags and environment variables.

For example, `--config ~/stern.yaml` will be expanded to like `--config /home/ubuntu/stern.yaml` by the shell. `export STERNCONFIG=~/stern.yaml` will also be expanded in the same way.

```
$ export STERNCONFIG=~/stern.yaml
$ echo $STERNCONFIG
/home/ubuntu/stern.yaml
```

Reply:
OK, I'll change to use `os.UserHomeDIr()`.